metadata-jobs/mae-consumer/src/main/java/com/linkedin/metadata/kafka/MetadataChangeLogProcessor.java

```diff
@@ -130,14 +136,35 @@ public void consume(final ConsumerRecord<String, GenericRecord> consumerRecord)
       } else {
         updateSearchService(entitySpec.getName(), urn, aspectSpec, aspect);
         updateGraphService(urn, aspectSpec, aspect);
+        updateSystemMetadata(event.getSystemMetadata(), urn, aspectSpec);
+      }
+    }
+    if (event.getChangeType() == ChangeType.DELETE) {
```

Comment:
nit: else if
Makes it more clear that these two if statements will never both run